### ISSUE TITLE
SC2054 false negative with only a single comma in array element

### ISSUE THREAD

**Issue by benpaxton-hf** [koalaman/shellcheck#1318]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2054
- My shellcheck version (`shellcheck --version` or "online"): 0.5.0
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [ ] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#! /bin/bash

nowarn=(
	abc,def
	ghi,jkl
)

warn=(
	abc,def,ghi
	jkl,mno,pqr
)

# use the arrays to avoid warnings for that
echo "${nowarn[@]}"
echo "${warn[@]}"
```

#### Here's what shellcheck currently says:
```
In test.sh line 8:
warn=(
     ^-- SC2054: Use spaces, not commas, to separate array elements.
```

#### Here's what I wanted or expected to see:
```
In test.sh line 3:
nowarn=(
     ^-- SC2054: Use spaces, not commas, to separate array elements.

In test.sh line 8:
warn=(
     ^-- SC2054: Use spaces, not commas, to separate array elements.
```

It looks like https://github.com/koalaman/shellcheck/blob/master/src/ShellCheck/Analytics.hs#L1212 should say `length (...) > 0`, rather than `length (...) > 1`.

**Comment by koalaman** [comment]
This check was indeed far too cautious, and it's now been fixed by @Gandalf- . Thanks!
